lib/server.js

```diff
@@ -68,6 +69,10 @@ exports.json = (req, {limit = '1mb'} = {}) => Promise.resolve().then(() => {
   })
 })
 
+exports.query = req => Promise.resolve().then(() => {
```

Comment:
Gotcha. Thought we might want to keep it the same as the `.json` API so made it use the promise. Makes sense to make it sync though.